### PR TITLE
Document jump to date labs feature (MSC3030)

### PR DESCRIPTION
**Pull request by MadLittleMods**
Document jump to date labs feature

Feature added in:

 - https://github.com/matrix-org/matrix-react-sdk/pull/7339
 - https://github.com/matrix-org/matrix-react-sdk/pull/7372
 - MSC3030: https://github.com/matrix-org/matrix-doc/pull/303

---
This PR currently has no changelog labels, so will not be included in changelogs.

A reviewer can add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is, or add `Type: [enhancement/defect/task]` to the description and I'll add them for you.

### CODE REVIEW

File: docs/labs.md [hunk 1]
```diff
@@ -21,6 +21,19 @@ Enables rendering of LaTeX maths in messages using [KaTeX](https://katex.org/).
...
+
+Note: This labs feature is only visible when your homeserver has MSC3030 enabled
+(in Synapse, add `experimental_features` -> `msc3030_enabled` to your
+`homeserver.yaml`) which means `GET /_matrix/client/versions` responds with
+`org.matrix.msc3030` under the `unstable_features` key.
+
+Adds a dropdown menu to the date separator headers in the timeline which allows
+you to jump to last week, last month, the beginning of the room, or choose a
+date from the calendar.
+ß
```
Comment (turt2live) [inline docs/labs.md:34]:
```suggestion

```

**Review by turt2live — APPROVED** [review]
otherwise lgtm

File: docs/labs.md [hunk 2]
```diff
@@ -21,6 +21,18 @@ Enables rendering of LaTeX maths in messages using [KaTeX](https://katex.org/).
...
+
+Note: This labs feature is only visible when your homeserver has MSC3030 enabled
+(in Synapse, add `experimental_features` -> `msc3030_enabled` to your
+`homeserver.yaml`) which means `GET /_matrix/client/versions` responds with
+`org.matrix.msc3030` under the `unstable_features` key.
+
+Adds a dropdown menu to the date separator headers in the timeline which allows
+you to jump to last week, last month, the beginning of the room, or choose a
+date from the calendar.
+Also adds the `/jumptodate 2022-01-31` slash command.
```
Comment (MadLittleMods) [inline docs/labs.md:34]:
```suggestion

Also adds the `/jumptodate 2022-01-31` slash command.
```

**Comment by MadLittleMods** [comment]
Thanks for the review @turt2live 🐍

**Pull request merged**